Gordon360/Models/ViewModels/BasicInfoViewModel.cs

```diff
@@ -93,5 +95,20 @@ public bool NicknameContains(string searchString)
         {
             return Nickname?.ToLower()?.Contains(searchString) ?? false;
         }
+
+        public bool MaidenNameMatches(string matchString)
+        {
+            return MaidenName?.ToLower() == matchString;
+        }
+
+        public bool MaidenNameStartsWith(string searchString)
+        {
+            return MaidenName?.ToLower()?.StartsWith(searchString) ?? false;
```

Comment:
```suggestion
            return MaidenName?.ToLower().StartsWith(searchString) ?? false;
```

Reply:
There can be no way for `ToLower` to be null at this point, so this is unnecessary.